src/effects/timer.js

```diff
@@ -0,0 +1,26 @@
+import { startTimer, stopTimer, tickTimer } from '../actions/game'
+
+class TimerEffect {
+  setup() {
+    this.interval = null
+  }
+
+  register() {
+    return {
+      [startTimer]: this.startTimer,
+      [stopTimer]: this.stopTimer
+    }
+  }
+
+  startTimer(repo) {
+    this.interval = setInterval(() => {
+      repo.push(tickTimer)
+    }, 1000)
```

Comment:
NBD, but what do you think of: `this.interval = setInterval(repo.prepare(tickTimer), 1000)`?